Agent architecture by Sergei Parfenov/index.md

```
@@ -0,0 +1,264 @@
+
+-   [https://arxiv.org/abs/2310.04406](https://arxiv.org/abs/2310.04406)
+
+-   [https://arxiv.org/abs/2305.04091](https://arxiv.org/abs/2305.04091)
+
+-   [https://arxiv.org/abs/2305.18323](https://arxiv.org/abs/2305.18323)
+
+-   [https://arxiv.org/abs/2312.04511](https://arxiv.org/abs/2312.04511)
+
+## Об авторе
+
+https://www.linkedin.com/in/sergeiparfenov/
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)